Sources/StrongContractResponse/All/URLQueuryItem.swift

```diff
@@ -4,14 +4,16 @@
 //
 //  Created by Scott Lydon on 4/6/24.
 //
-
 import Foundation
 
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
```

Comment:
Please test this in environments where we don't have access to the import I think it will throw an error that it can't find url something or other.  

I'm starting to think you decided to wrap imports with if else as a precaution not because you are addressing actual errors everytime.